src/proof/alethe/alethe_post_processor.cpp

```diff
@@ -443,6 +504,21 @@ bool AletheProofPostprocessCallback::update(Node res,
                            new_args,
                            *cdp);
     }
+    case ProofRule::THEORY_REWRITE:
+    {
+      ProofRewriteRule di;
+      if (rewriter::getRewriteRule(args[0], di))
```

Comment:
This rule is only generated internally when `rewriter::getRewriteRule` is expected to succeed, so we should translate this rule under this assumption. There is no need to treat the case where a hole would have to be produced, because this should be seen as a cvc5 error.

Reply:
Makes total sense, ~~I'll add an Assertion though just in case.~~